File: Projects-Solutions/Numbers/Java/Mortgage.java
/*Mortgage Calculator:
* Calculate the monthly payments of a fixed term mortgage over given Nth terms at a given interest rate. 
* Also figure out how long it will take the user to pay back the loan.
*/

import java.util.Scanner;

public class Mortgage {
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		double loan_amount, interest, interest_rate;
		int months;
		
		System.out.println("Amount loaned");
		loan_amount = input.nextDouble();
		
		System.out.println("Time in months");
		months = input.nextInt();
		
		System.out.println("Interest rate");
		interest_rate = input.nextDouble();
		
		interest_rate = (interest_rate/100);
		interest = loan_amount * interest_rate * months;
		
		System.out.println("Interest: " + interest);
		System.out.println("Total money owed: " + (interest + loan_amount));
		System.out.println("Minimum payment per: " + (interest + loan_amount)/months);
	}
}

File: Projects-Solutions/Numbers/Go/mortgage.go
/* Mortgage Calculator:
*  Calculate the monthly payments of a fixed term mortgage
*  over given Nth terms at a given interest rate.
*  Also figure out how long it will take the user to pay back the loan
 */

package main

import "fmt"

func main() {
	var months, loan_amount, interest_rate, interest float64

	fmt.Print("Amount of loan: ")
	fmt.Scanf("%f", &loan_amount)

	fmt.Printf("Time in months: ")
	fmt.Scanf("%f", &months)

	fmt.Printf("Interest rate: ")
	fmt.Scanf("%f", &interest_rate)

	interest_rate = (interest_rate / 100)

	interest = (loan_amount * months * interest_rate)

	fmt.Printf("\nInterest : %.2f", interest)
	fmt.Printf("\nTotal money owed : %.2f", (interest + loan_amount))
	fmt.Printf("\nMinimum payment per month:  %.2f\n", (interest+loan_amount)/months)
}
